### DISCUSSION TITLE
New iCub LIVE release 7.10

### DISCUSSION THREAD

**Discussion by mbrunettini** [robotology/community#369] [Releases]
Hi,
We are happy to announce a new release of iCub LIVE OS for the iCub PC104:

**iCub LIVE OS version 7.10**

This version is based on the latest Debian stable release 10 (codename "Buster") with upgraded system software and libraries, like:
- kernel 4.19.0-6-rt-amd64
- CMAKE 3.13 
- gcc 8.3

You can download it here:
[http://www.icub.org/download/pc104-images/](http://www.icub.org/download/pc104-images/)

Please follow the instructions on iCub wiki:
[http://wiki.icub.org/wiki/The_Linux_on_the_pc104](http://wiki.icub.org/wiki/The_Linux_on_the_pc104)

**Comment by pattacini** [comment]
cc @robotology/everyone
